criteo_distributed_training/autoscalar_aws.yml

```diff
@@ -97,12 +99,16 @@ available_node_types:
         # For more documentation on available fields, see:
         # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
         node_config:
-            InstanceType: c5.xlarge
+            InstanceType: c5ad.12xlarge
             ImageId: ami-02f3416038bdb17fb # Deep Learning AMI (Ubuntu) Version 30
             BlockDeviceMappings:
                 - DeviceName: /dev/sda1
                   Ebs:
-                      VolumeSize: 25
+                      VolumeSize: 150
+            # KeyName: MyEC2KeyPair
+            IamInstanceProfile:
+                Arn: arn:aws:iam::199696198976:instance-profile/ray-autoscaler-v1
```

Comment:
Could you add comments or link to doc, how other users can access these?

Reply:
I have added comments now.